src/libs/pool-data-provider/hooks/use-pool-data.tsx

```diff
@@ -94,30 +88,22 @@ export function usePoolData(
     setLoadingUserReserves(false);
   };
 
-  useEffect(() => {
-    setLoadingReserves(true);
-    setLoadingUserReserves(true);
-
-    if (!skip) {
-      fetchData();
-      const intervalID = setInterval(
-        () => fetchData(),
-        errorReserves || errorUserReserves ? RECOVER_INTERVAL : POOLING_INTERVAL
-      );
-      return () => clearInterval(intervalID);
-    } else {
-      setLoadingReserves(false);
-      setLoadingUserReserves(false);
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [currentAccount, lendingPoolAddressProvider, skip]);
+  usePolling(fetchReserves, POLLING_INTERVAL, skip, [skip, poolDataProviderAddress, network]);
```

Comment:
splitting this up allows us to a) have different intervals b) only refetch user reserves